### PR TITLE
Add blog post for Cake 2.0 RC2

### PR DESCRIPTION
**Pull request by augustoproiete**
Add blog post for Cake 2.0 RC2

### CODE REVIEW

File: input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+- [Cake v2.0.0-rc0001 release notes](/blog/2021/11/cake-v2.0.0-rc0001-released)
+
+In this blog post, we will go through what's new in Cake RC 2, compared to [RC 1](/blog/2021/11/cake-v2.0.0-rc0001-released).
+
+### Highlights of this release
+
+- Added support for .NET v6.0.100 RTM
+- Improvements and bug fixes to the new `dotnet` aliases
+- Updated .NET target frameworks
+- Impro
```
Comment (devlead) [inline input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md:23]: This seems a bit cut off?

**Review by devlead — CHANGES_REQUESTED** [review]
Left one minor comment otherwise LGTM 👍

File: input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+- [Cake v2.0.0-rc0001 release notes](/blog/2021/11/cake-v2.0.0-rc0001-released)
+
+In this blog post, we will go through what's new in Cake RC 2, compared to [RC 1](/blog/2021/11/cake-v2.0.0-rc0001-released).
+
+### Highlights of this release
+
+- Added support for .NET v6.0.100 RTM
+- Improvements and bug fixes to the new `dotnet` aliases
+- Updated .NET target frameworks
```
Comment (pascalberger) [inline input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md:22]: Above you mention that this blog post lists changes between RC 1 and RC2. We haven't changed target frameworks between these releases.

**Review by pascalberger — CHANGES_REQUESTED** [review]
One small thing. Otherwise LGTM

File: input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md (hunk 2 above)
Comment (augustoproiete) [reply input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md]: Thanks, removed it

File: input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md (hunk 1 above)
Comment (augustoproiete) [reply input/blog/2021-11-26-cake-v2.0.0-rc0002-released.md]: Thanks & fixed, meant to be `Improvements to IntelliSense support when using Visual Studio Code` (re: reference assemblies)

**Review by pascalberger — APPROVED** [review]
LGTM

**Review by devlead — APPROVED** [review]
LGTM 👍

**Pull request merged**